src/test/java/org/sagebionetworks/bridge/services/AccountServiceTest.java

```diff
@@ -263,6 +265,7 @@ public void createAccount() {
         assertEquals(createdAccount.getStatus(), UNVERIFIED);
         assertEquals(createdAccount.getMigrationVersion(), MIGRATION_VERSION);
         assertEquals(createdAccount.getNote(), TEST_NOTE);
+        assertEquals(createdAccount.getClientTimeZone(), TEST_CLIENT_TIME_ZONE);
```

Comment:
Need to add a check to the updateSuccess method to verify that this field is editable.